contracts/balancer/BConst.sol

```diff
@@ -20,6 +20,9 @@ contract BConst {
   // Minimum time passed between each weight update for a token.
   uint256 internal constant WEIGHT_UPDATE_DELAY = 30 minutes;
 
+  // Minimum time between each change to a token's minimum balance.
+  uint256 internal constant MIN_BAL_UPDATE_DELAY = 6 hours;
```

Comment:
This reorders storage, not a problem?

Reply:
It's a constant so it won't affect storage.